### PR TITLE
Export FPGA version und temperature in LDMRS interface

### PR DESCRIPTION
**Pull request by rostest**
Helperfunction to export FPGA version und temperature in LDMRS interface

### CODE REVIEW

File: src/devices/LD_MRS.cpp [hunk 1]
```diff
@@ -450,6 +450,25 @@ std::string LDMRS::getFirmwareVersion()
...
+{
+	if (m_lux == NULL)
+	{
+		return "(none)";
+	} else {
+		return m_lux->getFPGAVersion();
+	}
+}
+
+std::string LDMRS::getTemperature()
```
Comment (jspricke) [inline src/devices/LD_MRS.cpp:463]: why not return the double?

Comment (rostest) [reply src/devices/LD_MRS.cpp:463]: It's just for the diagnosis. But a double is as good or even better. Please do not hesitate to change the return value. Btw, thanks for the excellent work in libsick_ldmrs. It's a pleasure to use such a well structured project!

Comment (jspricke) [reply src/devices/LD_MRS.cpp:463]: it's your PR, so please go ahead ;).

Comment (rostest) [reply src/devices/LD_MRS.cpp]: Done :)

File: src/devices/LD_MRS.cpp [hunk 2]
```diff
@@ -460,13 +460,13 @@ std::string LDMRS::getFPGAVersion()
 	}
 }
 
-std::string LDMRS::getTemperature()
+double LDMRS::getTemperature()
 {
 	if (m_lux == NULL)
 	{
-		return "(none)";
+		return ::NaN_double; //"(none)";
```
Comment (jspricke) [inline src/devices/LD_MRS.cpp:467]: remove the comments

File: src/devices/LD_MRS.cpp [hunk 3]
```diff
@@ -460,13 +460,13 @@ std::string LDMRS::getFPGAVersion()
...
 	} else {
-		return std::to_string(m_lux->getTemperature());
+		return m_lux->getTemperature(); // std::to_string(m_lux->getTemperature());
```
Comment (jspricke) [inline src/devices/LD_MRS.cpp:469]: dito.

**Review by jspricke — CHANGES_REQUESTED** [review]

**Review by jspricke — APPROVED** [review]

**Comment by jspricke** [comment]
Thx!

**Comment by rostest** [comment]
Thanks a lot for the quick response and this excellent library!

**Pull request merged**